package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@honeybadger-io/react",
-  "version": "1.0.2",
+  "version": "2.0.0",
```

Comment:
I decided to bump the major version because we are introducing a breaking change. See details in PR description and we can discuss this further.